server/routes/bookedtoday/unexpectedArrivals/validation/searchForExistingRecordsValidation.ts

```diff
@@ -9,11 +9,15 @@ const SearchForExistingRecordsValidator: Validator = ({
   prisonNumber,
   pncNumber,
 }: Record<string, string>): ValidationError[] => {
-  if (firstName && !lastName && (!day || !month || !year)) {
+  if (firstName && !lastName && !day && !month && !year) {
     return [{ text: 'Enter a last name and a date of birth', href: '#last-name' }]
   }
 
-  if (!lastName && !prisonNumber && !pncNumber) {
+  if (firstName && !lastName) {
+    return [{ text: 'Enter a last name', href: '#last-name' }]
+  }
+
+  if ((!firstName && !lastName && day && month && year) || (!lastName && !prisonNumber && !pncNumber)) {
```

Comment:
can this can just be `(!lastName && !prisonNumber && !pncNumber)` ? 

Wouldn't the other case be covered anyway?

Reply:
yes it can be.

Reply:
No because it would allow a search using just dob plus either prison or pnc number

Reply:
cool, might be nice to have this as a parameterised test so we can see that every combo has been handled. 

there's no tests for valid cases for instance